guide/private-key-management/single-user-schemes.md

```diff
@@ -53,9 +53,9 @@ Now let's dive in and look at the various schemes that might be suitable for a p
 
 By using an automatic cloud backup for the private key the user is NOT required to manually backup anything when setting up a new [bitcoin wallet]({{ '/guide/glossary/#bitcoin-wallet' | relative_url }}). This is sometimes also called *seedless*, as the user is never exposed to the traditional *seed*, or [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) that is used to initiate a wallet.
 
-This scheme can be used for those new to bitcoin, or novices who would find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups. It could also be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds.
+This scheme is ideal for those new to bitcoin who may find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups, hot or spending wallets which store private keys locally and who's security model does not require a manual backup, or be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds.
```

Comment:
```suggestion
This scheme is ideal for those new to Bitcoin who may find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups, hot or spending wallets which store private keys locally and who's security model does not require a manual backup, or be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds.
```

Reply:
Good detail to add. This sentence should be broken up a bit though to improve readability.

<img width="604" alt="Screenshot 2021-05-11 at 04 34 47" src="https://user-images.githubusercontent.com/183140/117749556-43b4a600-b212-11eb-8d93-524c8859f82d.png">

Reply:
Let me know what you think of the change :)